src/main/java/org/apereo/openlrs/storage/elasticsearch/NormalizedElasticsearchTierTwoStorage.java

```diff
@@ -49,6 +49,10 @@ public OpenLRSEntity findById(String id) {
 	}
 
 	@Override
+	public OpenLRSEntity findBySourceId(String sourceId) { throw new UnsupportedOperationException(); }
+
+
+		@Override
```

Comment:
I'll try reformatting this file.  If the reformatting changes too much, I'll limit the formatting to the methods I was working with.  Then I'll commit the changes.